contracts/VotingToChangeKeys.sol

```diff
@@ -50,8 +51,9 @@ contract VotingToChangeKeys {
         _;
     }
 
-    function VotingToChangeKeys(address _proxyStorage) public {
+    function VotingToChangeKeys(address _proxyStorage, bool _demoMode) public {
```

Comment:
same thing. I don't think you should pass a bool
first reason - you can skip it and it defaults to false
second, it adds if/else logic